apps/pie-docs/src/components/chip/overview/overview.md

```diff
@@ -0,0 +1,326 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: Chip
+    order: 1
+shouldShowContents: true
+---
+
+## Overview
+
+The purpose of chips is to provide a visual representation of a specific entity or attribute, such as a selected option or a labelled category. Chips can also be interactive, allowing users to remove or modify the selected choices.
+
+Chips are commonly used in various contexts, including filtering options, search results or any situation where concise and visually distinct information needs to be displayed.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/overview.svg",
+    alt: "A group of chips that are horizontally stacked together."
+} %}
+
+---
+
+## Dos and Don’ts
+
+{% usage {
+    do: {
+        type: usageTypes.list,
+        items: [
+            "Chips should appear dynamically as a group of multiple interactive elements."
+        ]
+    },
+    dont: {
+        type: usageTypes.list,
+        items: [
+            "Don’t use when an interaction is not required, use a [tag](/components/tag/) if needed."
+        ]
+    }
+} %}
+
+___
+
+## Anatomy
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/anatomy.svg",
+    alt: "Anatomy of the chip component.",
+    width: 210
+} %}
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+        "**Icon (Optional):** Non-interactive icon that can be used to visually support the string.",
+        "**String:** Text label informing the user of the option/selection.",
+        "**Close (optional)**: Allows the Chip to be dismissible, but the application should provide a way for them to easily add it back."
+    ]
+} %}
+
+---
+
+## Variations
+
+### Default
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/variation-default.svg",
+    alt: "The default variant of the chip component.",
+    width: 63,
+    variant: "secondary"
+} %}
+
+### Outline
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/variation-outline.svg",
+    alt: "The outline variant of the chip component.",
+    width: 63
+} %}
+
+### Ghost
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/variation-ghost.svg",
+    alt: "The ghost variant of the chip component.",
+    width: 63
+} %}
+
+---
+
+## Modifiers
+
+### Icon
+
+Icons are always positioned to the left of the string.
```

Comment:
It would be more accurate to say that they are always in a "leading" position, and the alt text should reflect this too.

Reply:
@tingting-lau can you please advice here? ⭐

Reply:
updated

Reply:
updated